Userland/Applications/Spreadsheet/Tests/free-functions.js

```diff
@@ -203,3 +203,33 @@ describe("Lookup", () => {
         expect(reflookup("80", R`A0:A9`, R`B0:B9`, undefined, "nextlargest").name).toEqual("B9");
     });
 });
+
+describe("integer() function", () => {
+    test("undefined", () => {
+        expect(integer(undefined)).toBe(NaN);
+    });
+    test("null", () => {
+        expect(integer(null)).toBe(NaN);
+    });
+    test("NaN", () => {
+        expect(integer(NaN)).toBe(NaN);
+    });
+    test("object", () => {
+        expect(integer({})).toBe(NaN);
+    });
+    test("function", () => {
+        expect(integer(() => {})).toBe(NaN);
```

Comment:
These should just throw, we don't have to inherit JavaScript's stupid decisions.

Reply:
done